pylspclient/lsp_client.py

```diff
@@ -15,7 +15,16 @@ def __init__(self, lsp_endpoint: LspEndpoint):
         self.lsp_endpoint = lsp_endpoint
 
 
-    def initialize(self, processId, rootPath, rootUri, initializationOptions, capabilities, trace, workspaceFolders):
+    def initialize(
+        self,
+        processId: Optional[int] = None,
+        rootPath: Optional[str] = None,
+        rootUri: Optional[str] = None,
+        initializationOptions: any = None,
+        capabilities: dict = None,
```

Comment:
`capabilities: OPtional[dict] = None,`

Reply:
It is actually a required parameter, but some are optional parameters above this. This should ideally be reordered, but that would be a breaking change.